tracer/src/Datadog.Trace/Iast/Aspects/System.IO/FileAspect.cs

```diff
@@ -7,12 +7,13 @@
 
 using System;
 using Datadog.Trace.AppSec;
+using Datadog.Trace.ClrProfiler;
 using Datadog.Trace.Iast.Dataflow;
 
 namespace Datadog.Trace.Iast.Aspects;
 
 /// <summary> File class aspects </summary>
-[AspectClass("mscorlib,System.IO.FileSystem,System.Runtime", AspectType.RaspIastSink, VulnerabilityType.PathTraversal)]
+[AspectClass("mscorlib,System.IO.FileSystem,System.Runtime", InstrumentationCategory.IastRasp, AspectType.Sink, VulnerabilityType.PathTraversal)]
```

Comment:
Would have been nice to have had these changes in a separate PR, I think it would have made it easier to validate the changes, but 🤷‍♂️